### DISCUSSION TITLE
How to customize which organs that specific molecules can enter? (updated)

### DISCUSSION THREAD

**Discussion by DR-99** [Open-Systems-Pharmacology/Forum#1347] [Questions & Problems]
Hello again! I find myself stuck again with a simulation that has no errors however, there are no results for several critical parameters when running. Despite, following the detailed tutorial on adding compartments and double-checking that my passive transport block, MSV and PSV are all updated, the results show no arterial concentrations, no concentrations in my added compartment and a logarithmically increasing venous concentration. So, it seems that there is some blockage at my new compartment. Is there some parameter or calculation that I may have missed, which needs to be tweaked? 

If it helps, the added compartment is supposed to be a lumped compartment, so representative of several deleted organs. I have already tried to adjust the weigh and volumes accordingly, and ensure that the neighborhoods and the new compartment have the correct tags. So, since the venous concentrations are monotonically increasing, I am assuming that there is some issue with the passive transports, but I can't think of any other suspects. 

I appreciated your last answer to my question, and I hope that you can help me out once more.

**Comment by StephanSchaller** [comment]
Yes, it sounds like you have a mass balance issue (blood flows are not all considered / well connected). hard to tell from afar. If I may ask, what is the purpose of lumping the organs (performance should not be an issue these days?)?

**Reply by DR-99** [reply]
I am trying to model the oral administration of a combination drug where one of the components has poorly predicted distribution kinetics (predictions and parameter estimations result in 2-3 compartments, but observations show that it is clearly 1 compartmental). Therefore, I wanted to simplify the distribution kinetics by lumping the peripheral organs that are not involved in absorption into one compartment and then performing the parameter estimation. 

A previous version connected the additional compartment directly to the portal vein and to an elimination depot. This seemed to work well with the simulated concentrations for this compartment having a better correspondence to the observations. However, on second thought, I considered whether there is an appropriate comparison and decided that I needed to add the lumped compartment as an organ that is connected to the circulation, which is where I am now stuck as there are no errors to guide me.

I would appreciate your thoughts on this strategy. As I am new to Mobi and fairly new to PBPK modelling, I am interested in understanding the underlying mechanisms of these models. Thanks in advance!

**Reply by StephanSchaller** [reply]
This approach kind of misses the point of PBPK modeling. INstead, PBPK could be used here to evaluate why you observe a "1-compartmental-like" PK and what is needed in terms of ADME process parameterization to capture this. This will help you understand the unique properties of your compound and allow decision-making in a translational setting using the PBPK model.

**Reply by StephanSchaller** [reply]
I can recommend a bit of reading on the topic: https://www.wiley.com/en-us/Physiologically+Based+Pharmacokinetic+(PBPK)+Modeling+and+Simulations:+Principles,+Methods,+and+Applications+in+the+Pharmaceutical+Industry,+2nd+Edition-p-9781119497684

**Reply by DR-99** [reply]
Yes, I agree with your point and I appreciate the reference. However, this component of the combination drug has very little physicochemical and clinical data available, and since it is not responsible for the therapeutic response but simply maximises it, I thought it would be ok to simplify it while ensuring that the primary component has a full PBPK model. 

Even so, I would still like to take the opportunity and understand better how the mass transports are configured in Mobi. It seems that both the venous and arterial blood concentrations are dependent on the target and source blood flows respectively. Considering that I followed the recommendation to use an existing organ as a template, I still cannot fathom the reason for this error. I have also tried only adding the additional compartment with no deletions, as well as defining transports specifically for the new compartment using the pre-existing ones as a template (adjusting the TissueOrgan accordingly). Has something like this ever been encountered, where an organ seems to stop blood flow?

**Reply by DR-99** [reply]
Update: I have determined the issue was not with the passive transports but with the MSV. Although, I only excluded one molecule from circulating normally, it stopped the other molecules from circulating as well. (The other molecules were excluded from a single compartment). Can someone shed light on the reason for this, please? Also, is there a preferred way to specify circulation of different molecules? The alternative method that I am considering is to attach "Not in container" tags to the molecules in the MBB, however it seems like I would have to do it for each parameter or are there specific ones (eg. Concentration) that I can tag?
